### DISCUSSION TITLE
Crazy stuttering since updating

### DISCUSSION THREAD

**Discussion by tsbreuer** [ppy/osu#29964] [Q&A] [answered]
### Type

Performance

### Bug description

So i havent played in a while, and today i went and tried to play a bit of osu! and well my performance has been incredibly weird after updating. Mind you this is a 7950x3D and 3080 system, osu isnt exactly on the list of things i'd expect bad performance on.

So i'd normally troubleshoot whats going on but i honestly have no idea to where to start since my PC seems to be otherwise running perfectly fine in all other games.

As to what the performance issue is... I'm not entirely sure? its hard to describe other than i'm getting insane stuttering and both my draw and update are jumping to 30ms constantly (like multiple times a second)

Should be visible in the video, no idea whats going on.

Not sure if there's any easily available guide to troubleshoot lazer but i couldnt find one.

### Screenshots or videos

https://www.youtube.com/watch?v=jy_ptb9Kynk&feature=youtu.be

### Version

2024.906.2

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17088060/compressed-logs.zip)

**Comment by tsbreuer** [comment]
Should probably also mention that osu! stable runs perfectly fine, seems to be a lazer specific problem.

**Comment by cihe13375** [comment]
I once had similar issue and found two different solutions:
1) switch to opengl
2) go to nvidia control panel - 3d settings and add an override to osu!lazer and play with the settings there (I cannot tell exactly with setting fixed it though)

**Comment by bdach** [comment, marked as answer]
Judging from what the input thread appears to be doing on that video this is probably https://github.com/ppy/osu/issues/19960 or similar.
